fuel/datasets/base.py

```diff
@@ -145,7 +158,15 @@ def next_epoch(self, state):
         return self.reset(state)
 
     def close(self, state):
-        """Cleanly close the dataset e.g. close file handles."""
+        """
+        Cleanly close the dataset e.g. close file handles.
```

Comment:
Scrutinizer isn't complaining because it's skipping this file I think (see #159), but there should be no new line after `"""`